### ISSUE TITLE
Support ESLint 8.x

### ISSUE THREAD

**Issue by MichaelDeBoey** [webpack-contrib/eslint-webpack-plugin#118]
ESLint has released the first beta versions of v8 🎉
https://eslint.org/blog/2021/08/eslint-v8.0.0-beta.0-released
https://eslint.org/blog/2021/08/eslint-v8.0.0-beta.1-released

It would be awesome to have official ESLint 8 support. 👊
I'm happy to help where I can of course 🙂

**Comment by snitin315** [comment]
PR welcome :)

**Comment by jsg2021** [comment]
I don't think this project uses any of the removed/deprecated exports. For example, this uses the `ESLint` export already. As far as I know, it should already be compatible.

**Comment by jsg2021** [comment]
Yeah, I think the only thing needed is the peerDep to be extended with `^7.0.0 || >=8.0.0-beta.0 <9`

**Comment by MichaelDeBoey** [comment]
@jsg2021 The deprecated `CLIEngine` is also used

**Comment by alexander-akait** [comment]
@ricardogobbosouza hello, can you look?

**Comment by alexander-akait** [comment]
Looks like `stylish` was removed from `eslint`
